Makefile

```diff
@@ -39,6 +39,12 @@ else
     $(error Unsupported architecture)
 endif
 
+FEATURE ?=
+ifneq ($(FEATURE),)
```

Comment:
We should take `make FEATURE=none` into consideration.

Reply:
I have excluded `FEATURE =none` in GitHub Actions workflow. Or should I move the condition check to here?

Reply:
You don't have to exclude `FEATURE=none` in GitHub Actions since we always do `make check`. Just refine `Makefile` instead.